java/ql/src/semmle/code/java/PrintAst.qll

```diff
@@ -622,6 +624,33 @@ final class JavadocElementNode extends PrintAstNode, TJavadocElementNode {
   JavadocElement getJavadocElement() { result = jd }
 }
 
+/**
+ * A node representing the `Import`s of a `CompilationUnit`.
+ * Only rendered if there is at least one import.
+ */
+final class ImportsNode extends PrintAstNode, TImportsNode {
+  CompilationUnit cu;
+
+  ImportsNode() { this = TImportsNode(cu) }
+
+  override string toString() { result = "(Imports)" }
+
+  override ElementNode getChild(int childIndex) {
+    childIndex >= 0 and
```

Comment:
This conjunct looks superfluous. It is implied by `rank[childIndex](...)` - `rank` returns indices starting from 1.